src/components/Calculator.css

```diff
@@ -0,0 +1,53 @@
+.calculator {
+  display: grid;
+  min-height: 70vh;
+  width: 28rem;
+  margin: 20px;
+  margin-left: 750px;
+  justify-content: end;
+  grid-template-columns: repeat(4, 7rem);
+  grid-template-rows: minmax(6rem, auto) repeat(5, 5rem);
+  font-family: cursive, sans-serif;
+  border: 2px solid black;
+  filter: drop-shadow(5px 3px 3px #3f3e3e);
+}
+
+.screen {
+  grid-column: 1 / -1;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+button {
+  width: 100%;
+  padding: 10px;
+  background-color: rgb(243, 240, 240);
+  border: 1px solid rgb(151, 150, 150);
+  cursor: pointer;
+  font-size: 25px;
+}
+
```

Comment:
| @YoutLone Calculator | Given Design |
|-|-|
| ![image](https://github.com/YoutLone/math-magicians/assets/93039359/979fc9bd-7bdc-4678-81f1-cd7c8f2340ef) | ![image](https://github.com/YoutLone/math-magicians/assets/93039359/8dc075aa-68c4-4fc3-8321-cf2382977d94) |

- [ ] It would be nice if you make your calculator look identical to the given design to follow the project requirements 👍

1. Kindly make your screen less height and also remove the space between calculator screen and buttons to follow the given design 😊
2. Kindly make your orange buttons of the same color as in the given design 🤔